samples/word/10-content-controls/insert-and-change-checkbox-content-control.yaml

```diff
@@ -0,0 +1,246 @@
+order: 8
+id: word-content-controls-insert-and-change-checkbox-content-control
+name: Manage checkbox content controls
+description: 'Inserts, updates, retrieves, and deletes checkbox content controls.'
+host: WORD
+api_set:
+    WordApi: '1.7'
+script:
+    content: |
+        $("#insert-controls").on("click", () => tryCatch(insertCheckboxContentControls));
+        $("#toggle-control").on("click", () => tryCatch(toggleCheckboxContentControl));
+        $("#change-controls").on("click", () => tryCatch(toggleCheckboxContentControls));
+        $("#delete-control").on("click", () => tryCatch(deleteCheckboxContentControl));
+        $("#setup").on("click", () => tryCatch(setup));
+
+        async function insertCheckboxContentControls() {
+          // Traverses each paragraph of the document and places a checkbox content control at the beginning of each.
+          await Word.run(async (context) => {
+            let paragraphs = context.document.body.paragraphs;
+            paragraphs.load("$none"); // Don't need any properties; just start each paragraph with a content control.
+
+            await context.sync();
+
+            for (let i = 0; i < paragraphs.items.length; i++) {
+              let contentControl = paragraphs.items[i]
+                .getRange(Word.RangeLocation.start)
+                .insertContentControl(Word.ContentControlType.checkBox);
+            }
+            console.log("Checkbox content controls inserted: " + paragraphs.items.length);
+
+            await context.sync();
+          });
+        }
+
+        async function toggleCheckboxContentControl() {
+          // Toggles the isChecked property of the first checkbox content control found in the selection.
+          await Word.run(async (context) => {
+            const selectedRange = context.document.getSelection();
+            let selectedContentControl = selectedRange
+              .getContentControls({
+                types: [Word.ContentControlType.checkBox]
+              })
+              .getFirstOrNullObject();
+            selectedContentControl.load("id,checkboxContentControl/isChecked");
+
+            await context.sync();
+
+            if (selectedContentControl.isNullObject) {
```

Comment:
Doesn't seem like these checks are working as intended. If I don't have a content control in my selection, an exception is thrown in the console.